Godeps/Godeps.json

```diff
@@ -164,9 +164,13 @@
 			"ImportPath": "gopkg.in/yaml.v2",
 			"Rev": "eb3733d160e74a9c7e442f435eb3bea458e1d19f"
 		},
-        {
-            "ImportPath": "github.com/palantir/stacktrace",
-            "Rev": "78658fd2d1772b755720ed8c44367d11ee5380d6"
-        }
+    {
+      "ImportPath": "github.com/palantir/stacktrace",
+      "Rev": "78658fd2d1772b755720ed8c44367d11ee5380d6"
+    },
+    {
+      "ImportPath": "github.com/ghodss/yaml",
```

Comment:
Hey @pwplusnick, not sure how `github.com/ghodss/yaml` is different from `gopkg.in/yaml.v2`

Reply:
I can rewrite it, but it is used in a function I lifted from openwhisk-cli to do the swagger parsing. Ideally, we move that function to openwhisk-client-go and that dependency will get removed.